specification/messaging/kafka.md

```diff
@@ -1,23 +1,94 @@
-<!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Kafka
---->
-
-# Instrumenting Kafka
+# Semantic Conventions for Kafka
 
 **Status**: [Experimental][DocumentStatus]
 
-This document defines how to apply semantic conventions when instrumenting Kafka.
-
 <!-- toc -->
 
+- [Kafka span attributes](#kafka-span-attributes)
+  * [Example: Apache Kafka with Quarkus or Spring Boot Example](#example-apache-kafka-with-quarkus-or-spring-boot-example)
 - [Kafka Metrics](#kafka-metrics)
-- [Kafka Producer Metrics](#kafka-producer-metrics)
-- [Kafka Consumer Metrics](#kafka-consumer-metrics)
+  * [General Kafka Metrics](#general-kafka-metrics)
```

Comment:
I feel we can drop "Kafka" in sub headings since the page is already about Kafka. Less repetitive :)